src/app/shared/mangoQueries.ts

```diff
@@ -22,11 +22,21 @@ export function findAllDocuments(selector, query) {
   `);
 }
 
-export function findMultiDocuments(selector, query) {
+export function findMultiDocuments(selectors) {
```

Comment:
Satisifies with "AND" condition. But think to change for other operator as well . @lmmrssa , so if yes then we need to send the selectors value as a query format from the requested page.
eg;
selector": {
    "year": {
      "$gt": 2010
    }
  },
if we need >,<, ! operator then this function will not supported. So , for this we need to send the "selectors" value same as of query value.